src/pages/item/index.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import '../../App.css'
+import './item.css'
+import {ItemContext} from '../../context/items'
+import {BasketContext} from '../../context/basket'
+import ItemList from '../home/items-list'
+import _ from 'lodash'
+import {useParams} from 'react-router-dom'
+import ItemTemplate from '../../components/item-template/item-template'
+
+
+
+ 
+
+const Item = (props) => {
+
+  const basket = useContext(BasketContext)
+  const item = useContext(ItemContext)
+  
+  const itemPage = useContext(ItemContext)
+  const {id} = useParams()
+  const selection = _.find(ItemList, {id: itemPage.item && id} )
```

Comment:
can remove itemPage.item, if we have it in url params then we don't need it in context really